### PR TITLE
[#197] update FP with TE0 calcs

### LINKED ISSUE #197: TE0 calculations
Now that #192 was merged, we need to do the FP calcs with the updated code

### PR DESCRIPTION
**Pull request by jsadler2**
This PR is just running the updated code to calculate TE0 for all of the sites/replicates/sinks for tmmx.

Closes #197

### CODE REVIEW

File: 2a_model/src/models/2_multitask_dense/Snakefile [hunk 1]
```diff
@@ -17,7 +17,8 @@ rule all:
                  rep=list(range(config['num_replicates'])),
                  site_id=['01480870'],
                  year=[2012]),
-          f"{out_dir}/{config['exp_name']}_func_perf.csv"
+          f"{out_dir}/{config['exp_name']}_func_perf.csv",
+          f"{out_dir}/observed_func_perf.csv"
```
Comment (jsadler2) [inline 2a_model/src/models/2_multitask_dense/Snakefile:21]: I switched the "observed_func_performance" to this Snakefile b/c it only gets used once. The one in 0_baseline_LSTM also gets used by the 1_multitask models so the "observed_func_performance" was getting produced multiple times unnecessarily.

**Comment by jsadler2** [comment]
BTW - I checked these and everything looks unchanged except the TE0's. 

Screenshot 2023-02-20 at 11 48 35 AM

**Comment by galengorski** [comment]
> BTW - I checked these and everything looks unchanged except the TE0's.
> 
> Screenshot 2023-02-20 at 11 48 35 AM

That's great, thanks for sanity-checking that

**Review by galengorski — APPROVED** [review]
Looks great @jsadler2

**Pull request merged**